packages/block-library/src/gallery/block.json

```diff
@@ -111,6 +111,7 @@
 		"units": [ "px", "em", "rem", "vh", "vw" ],
 		"spacing": {
 			"blockGap": true,
+			"__experimentalSkipSerialization": true,
```

Comment:
Should we just specify `blockGap` here, in case we want to add in padding or other spacing at some point?

```suggestion
			"__experimentalSkipSerialization": [ "blockGap" ]
```

Also, I noticed that even without skipping serialization, it looks like the rules in this PR win out, but it's still neater to skip rendering the unneeded styles 👍

![image](https://user-images.githubusercontent.com/14988353/161476030-180fddbb-dc35-45ff-92b9-5064a4536509.png)

Reply:
have narrowed skipSerialization down to block gap as suggested - may avoid any future confusion for anybody playing with adding other spacing options